src/sentry/api/endpoints/organization_repositories.py

```diff
@@ -73,6 +73,19 @@ def post(self, request, organization):
             }, status=403)
 
         provider_id = request.DATA.get('provider')
+
+        if provider_id == 'integrations:github':
```

Comment:
what about making this more generic -- something like `if provider_id.startswith('integrations:'):` because we'll also probably have do use keys like that for bitbucket and vsts

Reply:
and can you also do an `internal-catchall` feature check like above